src/styles/block/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__image-content {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin: 40px 16px 0;
+    padding: 0;
+
+    &-text {
+      color: $primary-color;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      text-align: left;
+      margin-block: 0;
+    }
+  }
+
+  &__code {
+    display: flex;
+    margin-left: 16px;
+    margin-top: 4px;
+    height: 14px;
+    align-items: center;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: left;
+      font-weight: inherit;
```

Comment:
The font-weight property for '.card__code-text' is set to inherit, which might lead to inconsistency if the parent's font-weight changes. It's better to define a specific font-weight value here.